modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -712,7 +712,7 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
      *  @returns 4-dimansional Mat with NCHW dimensions order.
      */
     CV_EXPORTS_W Mat blobFromImage(const Mat& image, double scalefactor=1.0, const Size& size = Size(),
-                                   const Scalar& mean = Scalar(), bool swapRB=true, bool crop=true);
+                                   const Scalar& mean = Scalar(), bool swapRB=true, bool preprocess=false, bool crop=true);
```

Comment:
`blobFromImage` is already a kind of image preprocessing. So `preprocess=false` means do nothing I guess. I think it isn't necessary.